### PR TITLE
feat(cubestore-driver): Support persistent flag for query

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

File: packages/cubejs-query-orchestrator/src/orchestrator/QueryQueue.js [hunk 1]
```diff
@@ -750,7 +750,7 @@ export class QueryQueue {
 
       const activated = activeKeys && activeKeys.indexOf(this.redisHash(queryKey)) !== -1;
       if (!query) {
-        query = await queueConnection.getQueryDef(this.redisHash(queryKey));
+        query = await queueConnection.getQueryDef(queryKey);
```
Comment (ovr) [inline packages/cubejs-query-orchestrator/src/orchestrator/QueryQueue.js:753]: image
image

File: packages/cubejs-query-orchestrator/src/orchestrator/QueryQueue.js [hunk 2]
```diff
@@ -778,10 +778,9 @@ export class QueryQueue {
         );
         try {
           const handler = query?.queryHandler;
-          let target;
           switch (handler) {
             case 'stream':
-              target = this.getQueryStream(this.redisHash(queryKey));
+              const target = this.getQueryStream(queryKey);
```
Comment (ovr) [inline packages/cubejs-query-orchestrator/src/orchestrator/QueryQueue.js:783]: image

File: packages/cubejs-query-orchestrator/src/orchestrator/QueryQueue.js [hunk 3]
```diff
@@ -898,7 +897,7 @@ export class QueryQueue {
         });
         // closing stream
         if (query?.queryHandler === 'stream') {
-          const stream = this.getQueryStream(this.redisHash(queryKey));
+          const stream = this.getQueryStream(queryKey);
```
Comment (ovr) [inline packages/cubejs-query-orchestrator/src/orchestrator/QueryQueue.js:900]: image

File: packages/cubejs-query-orchestrator/src/orchestrator/QueryQueue.js [hunk 4]
```diff
@@ -770,18 +780,19 @@ export class QueryQueue {
...
+          () => queueConnection.updateHeartBeat(queryKeyHashed),
           this.heartBeatInterval * 1000
         );
         try {
           const handler = query?.queryHandler;
           switch (handler) {
             case 'stream':
-              const target = this.getQueryStream(queryKey);
-              await this.queryTimeout(this.queryHandlers.stream(query.query, target));
+              await this.queryTimeout(
```
Comment (ovr) [inline packages/cubejs-query-orchestrator/src/orchestrator/QueryQueue.js:793]: changed due to linter.

**Review by buntarb — APPROVED** [review]

**Pull request merged**